setup.py

```diff
@@ -1,12 +1,11 @@
 from setuptools import setup
-import datashader
 
 with open('requirements.txt') as f:
     install_requires = f.read().strip().split('\n')
 
 setup(name='datashader',
-      version=datashader.__version__,
```

Comment:
this doesn't even work if you don't have the dependencies installed.  If you wanted to go this route version should be stored in a single file (I don't know how to reconcile this with meta.yaml)